release-it.json

```diff
@@ -3,16 +3,15 @@
     "commitMessage": "chore: release v${version}",
     "tagName": "v${version}"
   },
-  "npm": {
-    "publish": true
-  },
-  "github": {
-    "release": true,
-    "releaseName": "v${version}"
-  },
+  "npm": false,
   "plugins": {
     "@release-it/conventional-changelog": {
       "preset": "angular"
+    },
+    "@release-it-plugins/workspaces": {
+      "workspaces": [
+        "packages/*"
+      ]
```

Comment:
Release it wasn't the best fit for me in one of my last projects, I just couldn't get it to work reliably in Monorepo. 

I used changesets: https://github.com/changesets/changesets which is pretty cool! But we can change it in another PR if this causes trouble

Reply:
Oh, good to know! I didn't know about that tool - definitely agree there are some rough edges for release-it! Happy to take a PR if it works for you. I copied this one from another project I set up, works okay, but it required some "R&D" earlier lol